### PR TITLE
don't harvest non-new low-TVL vaults (Link T-464)

### PR DESCRIPTION
**Pull request by AllTradesz**
- dont't harvest non-new low-TVL vaults (Link T-464)
- sync data files (Link T-464)

### CODE REVIEW

File: scripts/harvest_child.js [hunk 1]
```diff
@@ -327,19 +327,19 @@ const harvest = async (strat, harvesterPK, provider, options, nonce = null) => {
             };
           }
         } else {
-          tx = await stratContract.harvest(options);
+/**/      tx = await stratContract.harvest(options);
```
Comment (0xww) [inline scripts/harvest_child.js:330]: are this ```/**/``` added for some reason?

File: scripts/harvest_child.js [hunk 2]
```diff
@@ -515,15 +515,16 @@ const main = async () => {
...
-        // strats = stratsShouldHarvest.map(s => {
-        //   if (s.tvl < TVL_MINIMUM_TO_HARVEST) {
-        //     s.shouldHarvest = false;
-        //     s.notHarvestReason = `TVL is lower than min: ${TVL_MINIMUM_TO_HARVEST}`;
-        //   }
-        //   return s;
-        // });
-        // stratsFiltered = stratsFiltered.concat(strats.filter(s => !s.shouldHarvest));
-        // stratsShouldHarvest = strats.filter(s => s.shouldHarvest);
+				const i_TWO_WEEKS_AGO = new Date().getTime() / 1000 - 1209600;
```
Comment (0xww) [inline scripts/harvest_child.js:518]: could you use camelCase for variables name?

Comment (0xww) [reply scripts/harvest_child.js:518]: apply prettier (configuration is in .prettierrc file) to keep code clean in spaces

**Comment by AllTradesz** [comment]
stale, closing

**Pull request closed without merging**